cmd/worker.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"log"
+
+	"github.com/ScoreTrak/ScoreTrak/pkg/queue"
+	"github.com/spf13/cobra"
+)
+
+// workerCmd represents the worker command
+var workerCmd = &cobra.Command{
```

Comment:
How does worker ingest config?

Reply:
ah, never mind, seems like init from root will run. Will need to test this

Reply:
The worker command injests the config via the global var in the root.go. This is how viper recommends to pass the config